apps/root/src/pages/token-profile/token-distribution/index.tsx

```diff
@@ -91,29 +89,12 @@ const StyledBullet = styled.div<{ fill: string }>`
   `}
 `;
 
-const BorderLinearProgress = styled(LinearProgress)<{ fill?: string }>(
-  ({
-    theme: {
-      palette: { mode },
-      spacing,
-    },
-    fill,
-  }) => ({
-    background: colors[mode].background.primary,
-    height: spacing(2),
-    [`& .${linearProgressClasses.bar}`]: {
-      borderRadius: spacing(3),
-      background: fill,
-    },
-  })
-);
-
 const DATA_KEY_TO_LABEL: Record<
   keyof ReturnType<typeof useNetWorth>['assetsTotalValue'],
   ReturnType<typeof defineMessage>
 > = {
-  dca: defineMessage({ description: 'token-profile.distribution.dca', defaultMessage: 'DCA' }),
-  wallet: defineMessage({ description: 'token-profile.distribution.wallet', defaultMessage: 'Wallet' }),
+  dca: defineMessage({ description: 'token-profile.distribution.dca', defaultMessage: 'Recurring Investments' }),
```

Comment:
We generally dont need to change this since it can be later updated through localazy, its fine for now just future reference

Reply:
I agree with you, but we received feedback based on this, and I was able to find out that the component wasn't prepared for the real wording we are going to be using in production